### ISSUE TITLE
yarn create with typescript cause error

### ISSUE THREAD

**Issue by Suprhimp** [pmndrs/react-three-next#119]
I test with without command `-ts` or `--typescript`, it works well.

When I try to create with typescript it cause this error.

```
 > Success! Folder and files created for pathTraceTest 

> Installing packages... 

> Success! Installed dependencies for pathTraceTest 

> Installing packages for TypeScript... 

> Success! Installed dependencies for TypeScript 

> Initializing for TypeScript... 

> Success! Succeed to initialize for TypeScript 

node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[Error: ENOENT: no such file or directory, scandir '/Users/planningo/planningo/pathTraceTest/app'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: '/Users/planningo/planningo/pathTraceTest/app'
}

Node.js v18.0.0
error Command failed.
Exit code: 1
Command: /usr/local/bin/create-r3f-app
Arguments: next pathTraceTest tailwind -ts
Directory: /Users/planningo/planningo
Output:


```

I tried install node-sass and npm rebuild node-sass but it wasn't helped
when I look generated project it has no /app directory also.(I think it is not needed right?)

**Comment by blairjordan** [comment]
Cursory glance [here](https://github.com/utsuboco/create-r3f-app/blob/acbfd0829f1fc41691934624f35e779a70e57ece/lib/stacks/next.js#L326)and [here](https://github.com/utsuboco/create-r3f-app/blob/acbfd0829f1fc41691934624f35e779a70e57ece/lib/stacks/next.js#L355) reveals a possible source of the issue.

**Comment by RenaudRohlinger** [comment]
Thanks! Just published a patch with this fix on create-r3f-app.

https://github.com/utsuboco/create-r3f-app/commit/320feb79ac293aab49f4419b7370b2238c37dc82

Should also fix #125
